lib/log.js

```diff
@@ -0,0 +1,94 @@
+var path = require('path')
+var fs = require('fs')
+var Console = require('console')
+var mkdir = require('mkdirp')
+var wsock = require('websocket-stream')
+var http = require('http')
+var through = require('through2')
+
+var nodeLogger = null
+var browserLogger = null
+
+var PORT = 17509
```

Comment:
Agreed; this'd be ideal. However, it means you'll some kind of sideband to communicate to the browser process what port the node process chose. We could use IPC, but then we need to tell each new browser window about the port. Hardcoding, for now, seems like the least painful option.